### PR TITLE
Create ci.yml

### PR DESCRIPTION
**Pull request by abdul-moize**
This ci file contains the following checkers to make sure the files are up to standard
1. flake8
2. black
3. pylint
4. isort

### CODE REVIEW

**Comment by arslanashraf7** [comment]
The CI checks are failing, pylint is erroring out it's complaining about __init__ file, can you fix that or ignore the warning otherwise if a fix is not possible?

**Comment by abdul-moize** [comment]
> The CI checks are failing, pylint is erroring out it's complaining about **init** file, can you fix that or ignore the warning otherwise if a fix is not possible?

i tried ignoring/disabling the warning but it just stopped the working of pylint. therefore i added 3 paths *.py for root directory files, */*.py for files inside a subdirectory and */*/*.py for files inside subdirectory of subdirectory. i searched the net but couldnt find a better solution.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Lint with isort
+      run: |
+        isort . -c
+    - name: Lint with Pylint
+      run: |
+        pylint *.py */.py */*/*.py --disable=F0001
```
Comment (arslanashraf7) [inline .github/workflows/ci.yml]: Let's try `./*/**.py` and see if the pylint runs on all the .py files in all the directories. Also, Try to add `--disable=F0001` in a `.pylintrc` configuration file.

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]

**Review by arslanashraf7 — APPROVED** [review]

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[MESSAGES CONTROL]
+disable=F0001
+[MASTER]
+ignore=venv/, .idea/
```
Comment (arslanashraf7) [inline .pylintrc]: Just add a new line at the end of file.

**Review by arslanashraf7 — APPROVED** [review]
Looks good 👍 just a minor comment after which you can merge. Please do a squash merge.

**Pull request merged**